src/test/kotlin/com/zepben/evolve/cim/iec61968/customers/CustomerTest.kt

```diff
@@ -23,13 +23,13 @@ internal class CustomerTest {
     @Test
     internal fun hasEndDevicesReflectsNumEndDevices() {
         val customer = Customer()
-        assertThat("Customer has no end devices by default", not(customer.hasEndDevices()))
```

Comment:
This was using the `assertThat(actual: T, matcher: Matcher<T>)` method, which would pass even if `customer.hasEndDevices()` returned true. Also fixed other cases of this throughout the repo.